snuba/settings/__init__.py

```diff
@@ -28,6 +28,16 @@
 # Clickhouse Options
 CLICKHOUSE_MAX_POOL_SIZE = 25
 
+SENTRY_LOGICAL_PARTITIONS = 256
```

Comment:
Is there any consequence of this value being 256 for ST or Self hosted? I feel like slicing is something we would need only for SaaS, so it might be better to keep this as 1 for defaults and override in our prod settings file.

Reply:
The *logical* partitions should probably be 256 (or 512 or whatever we chose) everywhere IMO. They exist to make migration between different numbers of *physical* partitions easier. I set `LOCAL_PHYSICAL_SLICES` to 1 for the reason you state.

Arguably this should be a constant rather than a setting, it should essentially never change. But I wasn't sure where that would belong

Reply:
@onewland Agreed, please don't put this in the settings file. It's very important that 256 is constant and never overridden. Settings exists for things that need to be configured by users.

Reply:
moved that out of settings, let me know what you think